src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -240,4 +242,21 @@ public String getId() throws AbortException {
         }
     }
 
+    @Whitelisted
+    public List<Cause> getCauses() throws AbortException {
+        return build().getCauses();
+    }
+
+    @Whitelisted
+    public List<String> getCauseDescriptions() throws AbortException {
```

Comment:
Would rather not expose this as it encourages scripts to rely on inherently unreliable localized messages.